tubesync/sync/youtube.py

```diff
@@ -35,6 +35,35 @@ def get_yt_opts():
         opts.update({'cookiefile': cookie_file_path})
     return opts
 
+def get_channel_image_info(url):
```

Comment:
One final comment here, this info is almost certainly in the metadata stored in Media.metadata - you can likely just load it straight out of the JSON without having to invoke `yt-dlp` at all.